.circleci/config.yml

```diff
@@ -29,6 +32,42 @@ commands:
           name: Login to Quay
           command: docker login -u=$QUAY_USERNAME -p=$QUAY_PASSWORD quay.io
 
+  install-e2e-dependencies:
+    description: Install E2E Tests Dependencies
+    steps:
+      - run:
+          name: Install Dependencies
+          command: |
+            pip install -e ./test/e2e
+            pip install https://s3-us-west-2.amazonaws.com/get-cortex/master/python/cortex-master.tar.gz
+
+  run-e2e-tests:
+    description: Creates a temporary cluster and runs the cortex E2E tests
+    parameters:
+      provider:
+        description: cloud provider
+        type: enum
+        enum: [gcp, aws]
+      config:
+        description: cluster config file path
+        type: string
+        default: ./cluster.yaml
+    steps:
+      - run:
+          name: Create Cluster
+          command: cortex cluster up --config << parameters.config >> --configure-env << parameters.provider >> -y
```

Comment:
I'm thinking it might be good to explicitly use AWS credentials, e.g. for AWS we could set env vars like `NIGHTLY_AWS_ACCESS_KEY_ID` and `NIGHTLY_AWS_SECRET_ACCESS_KEY`, what do you think? I'm not as familiar with the best way to do this for GCP.

Reply:
sure, I can do that.

For GCP, we need to save the whole JSON file as an env var, and then create a file from its content